backend-core-data-impl/src/main/java/com/flowingcode/backendcore/dao/jpa/ConstraintTransformerJpaImpl.java

```diff
@@ -44,6 +44,11 @@
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 
+/**
+ * JPA implementation of {@link com.flowingcode.backendcore.model.ConstraintTransformer} for converting model constraints into JPA {@link jakarta.persistence.criteria.Predicate} instances.
```

Comment:
The following identifiers must never be linked and should always be wrapped in {@code}:
- The names of the direct supertype and any implemented interfaces of the documented type (in a type-level comment)
https://github.com/FlowingCode/DevelopmentConventions/blob/94fe5e464b0d91cb56542aeb098304c92f88b2ec/code-style.md#when-to-use-code